frontend/tutor-milton/src/components/googleLogin.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react"
+import firebase from "../config/firebase-config"
+import StyledFirebaseAuth from "react-firebaseui/StyledFirebaseAuth"
+
+
+class googleLogin extends Component {
+  state = { isSignedIn: false }
+  uiConfig = {
+    signInFlow: "popup",
+    signInOptions: [
+      firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+    ],
+    callbacks: {
+      signInSuccess: () => false
+    }
+  }
+
+  componentDidMount = () => {
+    firebase.auth().onAuthStateChanged(user => {
+      this.setState({ isSignedIn: !!user })
+      console.log("user", user)
+    })
+  }
+
+  render() {
+    return (
+      <div className="container1">
+        {this.state.isSignedIn ? (
+          <span>
+            <div>Signed In!</div>
```

Comment:
Export these to separate components each with properties so that it's easier to modify your code later.

Reply:
What I mean is that there should be a component displayed when you're signed in and a component for when you're not.